packages/flutter/lib/src/painting/inline_link_span.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show TextDecoration;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+
+import 'text_span.dart';
+import 'text_style.dart';
+
+/// An inline, interactive text link.
+class InlineLinkSpan extends TextSpan {
```

Comment:
Do we need to expose this? It's just a set of default values. For developers this seems easy enough to replicate even if it's made private?

Reply:
Also I wish these links change color once they're visited.

Reply:
Yeah good point, I'll make it private.

And changing color after visit is a great idea I didn't think of...  I guess I'll keep an eye out for requests for this after this PR is merged.  I'm not sure how people would expect this to work exactly.  I created https://github.com/flutter/flutter/issues/133718.

It would probably be possible for users to do this on their own with a TextLinker, but it wouldn't be very slick.